### ISSUE TITLE
Visualizer failed to read the sample trace file

### ISSUE THREAD

**Issue by kenmasumitsu** [google/mediapipe#849]
I followed the instruction at https://google.github.io/mediapipe/tools/tracing_and_profiling.html#analyzing-the-logs.

But Visualizer failed to read the sample trace file with the error below.

```
Loader
Could not load trace 'sample_trace.binarypb'.
Uploader
Could not load trace 'sample_trace.binarypb': Failure: Failed to read varint, encoding is invalid.
```

**Comment by mgyong** [comment]
@kenmasumitsu Can you provide more info?
1) what is the trace files generated for what devices?
2) Provide link to the sample trace file

**Comment by kenmasumitsu** [comment]
It is the file attached at https://google.github.io/mediapipe/data/visualizer/sample_trace.binarypb

**Comment by hadon** [comment]
Thanks for the report Kenmasu.  We found that the problem was with our source code release script, it was applying text transformations to sample_trace.binarypb, which obviously corrupted its binary format.  The release script is now fixed, and sample_trace.binarypb should be usable with the next release after MediaPipe v0.7.6.  I've attached the corrected file below in case it's helpful.  

[sample_trace.binarypb.zip](https://github.com/google/mediapipe/files/4960351/sample_trace.binarypb.zip)
